CityOfDetroit/bloom
# Change bedrooms to number of persons in questionnaire

## Issue

- Closes # (issue)
- Addresses #449 

## Description

Updated questionnaire to include "Persons" page in-lieu of "Bedrooms" page. Removed Bedrooms references and updated Cypress test to include new page.
![image](https://user-images.githubusercontent.com/82653098/131711357-690e23c9-b4b9-4b15-afda-0a1821796169.png)


## Type of change


## How Can This Be Tested/Reviewed?
Travel to questionnaire on mobile or desktop and notice refresh of second question.


## Checklist:
